test/reporter-html/window-onerror-preexisting-handler.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit HTML Reporter - No Markup</title>
+	<link rel="stylesheet" href="../../dist/qunit.css">
+	<script>
+		/* exported onerrorReturnValue, onerrorCallingContext */
```

Comment:
Might be able to remove "onerrorReturnValue" from exported comment here, will check.